pkg/api/types.go

```diff
@@ -46,7 +46,7 @@ type APIVersions struct {
 // Scanner interface that all scanners should define.
 type Scanner interface {
 	// Scan will scan the image
-	Scan(string, *docker.Image) error
+	Scan(path string, image *docker.Image) error
```

Comment:
maybe just a nit but I think we want something like *ScanOptions here so we can extended it in the future with more options?